src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -149,9 +149,28 @@ export const JobLogsContainer = props => {
           return intl.formatNumber(record.progress?.total);
         },
         errors: record => {
-          const failed = record.progress?.failed;
-
-          return failed || '';
+          const failedSrs = record.progress?.failed?.duplicatedSrs;
+          const failedOther = record.progress?.failed?.otherFailed;
+
+          switch (true) {
+            case failedSrs === 0 && failedOther > 0:
+              return `${failedOther}`;
+            case failedSrs > 0 && failedOther > 0:
+              return intl.formatMessage({
+                id: 'ui-data-export.column.errors.duplicatesWithOthers',
+              }, {
+                failedOther,
+                failedSrs,
+              });
+            case failedSrs > 0 && failedOther === 0:
+              return intl.formatMessage({
+                id: 'ui-data-export.column.errors.duplicates',
+              }, {
+                failedSrs,
+              });
```

Comment:
Why do you need "failedSrs" param here if translation does not contain it?
Also, I do not see any reason to have a translation for "column.errors.duplicates" if it is an empty string.

Reply:
also fixed now